resources/options_lookup.tsv

```diff
@@ -9793,22 +9793,22 @@ HVAC Heating Efficiency	"Fuel Wall/Floor Furnace, 68% AFUE"	ResStockArguments	he
+HVAC Heating Efficiency	"MSHP, SEER 16, 9.2 HSPF, Separate Backup, Max Load"	ResStockArguments	heating_system_type=none	heating_system_heating_efficiency=0	heating_system_heating_capacity=auto	heating_system_heating_autosizing_limit=auto	heating_system_fraction_heat_load_served=1	heating_system_has_flue_or_chimney=auto	heat_pump_type=mini-split	heat_pump_heating_efficiency_type=HSPF	heat_pump_heating_efficiency=9.2	heat_pump_cooling_efficiency_type=SEER	heat_pump_cooling_efficiency=16	heat_pump_sizing_methodology=MaxLoad	heat_pump_backup_sizing_methodology=auto	heat_pump_heating_capacity=auto	heat_pump_heating_autosizing_limit=auto	heat_pump_fraction_heat_load_served=1	heat_pump_cooling_capacity=auto	heat_pump_cooling_autosizing_limit=auto	heat_pump_fraction_cool_load_served=1	heat_pump_backup_type=separate	heat_pump_backup_fuel=electricity	heat_pump_backup_heating_efficiency=1	heat_pump_backup_heating_capacity=auto	heat_pump_backup_heating_autosizing_limit=auto	heat_pump_heating_capacity_retention_fraction=0.5	heat_pump_heating_capacity_retention_temp=5	heat_pump_is_ducted=false	heat_pump_backup_heating_lockout_temp=-20	heat_pump_compressor_lockout_temp=auto	heat_pump_cooling_compressor_type=variable speed	heat_pump_cooling_sensible_heat_fraction=auto	heat_pump_crankcase_heater_watts=auto	geothermal_loop_configuration=none	geothermal_loop_borefield_configuration=auto	geothermal_loop_loop_flow=auto	geothermal_loop_boreholes_count=auto	geothermal_loop_boreholes_length=auto	geothermal_loop_boreholes_spacing=auto	geothermal_loop_boreholes_diameter=auto	geothermal_loop_grout_type=auto	geothermal_loop_pipe_type=auto	geothermal_loop_pipe_diameter=auto
```

Comment:
My fault. I Moved the -20F from compressor lockout to backup heating lockout here. Good news is that this doesn't affect any releases.

https://github.com/NREL/resstock/issues/1397